config/database.yml.example

```diff
@@ -1,17 +1,19 @@
-# SQLite version 3.x
+#   SQLite version 3.x
```

Comment:
Why are you adding extra spaces?

Reply:
I thought that uniform indentation is good.